data/tsv/eng_latn_phonemic.tsv

```diff
@@ -1,5 +1,3 @@
-$deity	d e ɪ ɪ t i
```

Comment:
Based on a conversation with Cameron about `test_summary.py`, I noticed that I completely overlooked something. `eng_us_phonemic.tsv` has been renamed `eng_latn_phonemic.tsv`, the dialect information has been dropped. We've never had to handle a language having both dialect and script specifications and `split.py` is not setup to handle it (nor is [`_handle_wiki_name()`](https://github.com/kylebgorman/wikipron/blob/master/data/src/generate_tsv_summary.py#L17) in `generate_tsv_summary.py`).

This line overwrites the dialect information:
https://github.com/kylebgorman/wikipron/blob/c6c3b3f66dc2d85b0e0773000f8c7838fd40bf25/data/src/split.py#L48

This issue relates to #365 and I'm not really sure Cameron should have to fix it, but we'll need a fix for it before Cameron can run `generate_tsv_summary.py` and pass `test_summary.py`.